src/asha_bt.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <cstdint>
+#include "btstack.h"
+
+#include "hearing_aid.hpp"
+
+namespace asha {
+
+enum class ScanState {
+    Stop,
+    Scan,
+    IdentityResolving,
+    Connecting,
+    Disconnecting,
+    ServiceDiscovery,
+    CharDiscovery,
+    ReadROP,
+    ReadPSM,
+    SubscribeASPNotification,
+    Finalizing,
+    Complete,
+};
+
+constexpr uint16_t buff_size_sdu = 161;
+
+/* Struct to hold data from a GAP advertisment report */
+struct AdvertisingReport {
+    bd_addr_t address;
+    uint8_t   type;
+    uint8_t   event_type;
+    uint8_t   address_type;
+    uint8_t   rssi;
```

Comment:
rssi is actually an int8_t. The values I have observed usually sit from -100 to about -30 or so.